lib/shippo.rb

```diff
@@ -1,4 +1,4 @@
-#
+VERSION = '2.0.5'#
```

Comment:
@kigster thanks for noticing. It's actually what I did in my initial commit. I'll update it again.